public/src/components/manual-grading/ReviewForm.tsx

```diff
@@ -10,14 +10,13 @@ import ReviewResult from "../ReviewResult"
 const ReviewForm = (): JSX.Element => {
     const state = useAppState()
     const actions = useActions()
-    const courseID = getCourseID()
 
-    if (!state.activeSubmissionLink) {
+    if (!state.currentSubmission) {
```

Comment:
Maybe the following could be made into a method on `state`:
```ts
    if (!state.currentSubmission) {
        return <div>No submission selected</div>
    }
    if (!state.selectedAssignment) {
        return <div>No assignment Selected</div>
    }
```
Or similar. It seems this could be useful elsewhere as well. Not sure if we could return one or both of `assignment` and `submission`